src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -929,6 +955,9 @@ namespace cryptonote
       uint64_t start_time;
       uint64_t free_space;
       bool offline;
+      std::string bootstrap_daemon_address;
```

Comment:
Order is incorrect, should be `untrusted`, `bootstrap_daemon_address `, `height_without_bootstrap `

Reply:
There's no "correct" order.